django/src/rdwatch_scoring/urls.py

```diff
@@ -0,0 +1,12 @@
+from django.urls import path
+from rest_framework import routers
+
+from rdwatch_scoring.api import api
+
+router = routers.SimpleRouter(trailing_slash=False)
+
+urlpatterns = [
+    path('/api/scoring', api.urls),
+]
+
+urlpatterns += router.urls
```

Comment:
DRF router is not needed as we're only using ninja in these endpoints.
```suggestion
from django.urls import path

from rdwatch_scoring.api import api

urlpatterns = [
    path('/api/scoring', api.urls),
]
```